receiver/postgresqlreceiver/README.md

```diff
@@ -12,13 +12,13 @@
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 <!-- end autogenerated section -->
 
-This receiver queries the PostgreSQL [statistics collector](https://www.postgresql.org/docs/9.6/monitoring-stats.html).
+This receiver queries the PostgreSQL [statistics collector](https://www.postgresql.org/docs/12/monitoring-stats.html).
 
 > :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
 
 ## Prerequisites
 
-This receiver supports PostgreSQL versions 9.6+
+This receiver supports PostgreSQL versions 12.20+
```

Comment:
This should go into more details on the supported version, I think `12.20` is somewhat miss leading in the sense that we could be backwards compatible up to 12.20.

Reply:
Maybe we shouldn't really mention a version but only link to the latest supported upstream version then?

Reply:
Rephrased this sentence with a link to PG versions documentation